src/pages/SettingsPage.js

```diff
@@ -94,6 +120,29 @@ const SettingsPage = ({
                             Sign Out
                         </Text>
                     </TouchableOpacity>
+                    <View style={[styles.settingsPageBody, styles.mt6]}>
+                        <Text style={[styles.settingsPageHeading, styles.mt2, styles.mb4]} numberOfLines={1}>
```

Comment:
For this one, I apologize for the late change but could we actually just do `styles.textLabel, styles.textStrong` instead of adding a new style for `settingsPageHeading`? Also, I think we could make the bottom margin here a bit smaller, maybe try `styles.mb2` and see how that feels.

Reply:
And thinking on this some more, maybe we can just make this label say "Chat List Mode"? And then kill the text that immediately follows since we have small helper text that appears below the select menu.